system-tests/README.md

```diff
@@ -1,122 +1,260 @@
+```
+
+_Note for Windows PowerShell, the following command should be used:_
+
+```powershell
+./gradlew publishToMavenLocal -P"skip.signing"
+```
 
-### Building MVD project
+<br />
 
-MVD dependencies are Eclipse DataSpaceConnector(EDC) and Registration Service. Both of these dependencies are not published to any central artifactory yet so in local
-development we have to use locally published dependencies, once this is done MVD can be build using
+### Registration Service
+
+<br />
+
+Execute the following command from `Registration Service` root folder:
+
+```bash
+./gradlew publishToMavenLocal
+```
+
+<br />
+
+### MVD
+
+<br />
+
+Now that the publishing to the local repositories has been completed, `MVD` can be built by running the following command from the root of the `MVD` project folder:
 
 ```bash
 ./gradlew build -x test
 ```
 
-#### Publish EDC Registration Service and Identity Hub to local Maven
+## Local Test Execution
 
-Checkout [Eclipse DataSpaceConnector repository](https://github.com/eclipse-dataspaceconnector/DataSpaceConnector).
+- `MVD` system tests can be executed locally against a local `MVD` instance. 
+- `MVD` runs three `EDC Connectors` and one `Registration Service`.
 
-Publish EDC libraries to local Maven artifactory by executing gradle command `./gradlew publishToMavenLocal -Pskip.signing` from EDC root
-folder. On windows powershell command `./gradlew publishToMavenLocal -P"skip.signing"` can be used.
+_Note: Ensure that you are able to build `MVD` locally as described in the previous [section](#mvd)._
 
-Checkout [Registration Service repository](https://github.com/eclipse-dataspaceconnector/RegistrationService). 
+First, we need to build `EDC Connector`, `Identity Hub` and the `Registration Service`.
 
-Publish Registration Service libraries to local Maven artifactory by executing gradle command `./gradlew publishToMavenLocal` from Registration Service root folder.
+From the `MVD` root folder, execute the following command:
 
-Checkout [Identity Hub repository](https://github.com/eclipse-dataspaceconnector/IdentityHub).
+```bash
+./gradlew -DuseFsVault="true" :launcher:shadowJar
+```
 
-Publish Identity Hub libraries to local Maven artifactory by executing gradle command `./gradlew publishToMavenLocal` from Identity Hub root folder.
+From the `Registration Service` root folder, execute the following command:
 
-### Running test locally
+```bash
+./gradlew :launcher:shadowJar
+```
 
-MVD System tests can be executed locally against a local MVD instance. MVD runs three EDC Connectors and one Registration Service.
+From the `MVD` root folder execute the following commands to set the `Registration Launcher` path environment variable and start `MVD` using the `docker-compose.yml` file.
 
-First please make sure that you are able to build MVD locally as described in [Building MVD project](#building-mvd-project) section.
+```bash
+export REGISTRATION_SERVICE_LAUNCHER_PATH=/home/user/RegistrationService/launcher
```

Comment:
should we explain that the path should be adapted

Reply:
can resolve in next PR for the document as there are a number of other changes coming in to reflect MVD changes